.circleci/config.yml

```diff
@@ -519,7 +519,8 @@ jobs:
 
                     set -e
                     INTEGRATION_ID=$(uuidgen)
-                    kubectl create secret generic snyk-monitor -n snyk-monitor --from-literal=integrationId=${INTEGRATION_ID} --from-literal=dockercfg.json={}
+                    SERVICE_ACCOUNT_API_TOKEN=$(uuidgen)
```

Comment:
this is not uuid and also will need to be valid in order for the requests to be accepted by k-u - not sure what the implications here are

Reply:
Here we are only creating the SERVICE_ACCOUNT_API_TOKEN for starting the kubernetes monitor, but not really doing anything with it, therefore we can use a random uuid.